kommunicate/src/main/java/com/applozic/mobicomkit/api/conversation/SyncCallService.java

```diff
@@ -117,7 +118,7 @@ public synchronized void syncMessages(String key, Message message) {
                 Intent intent = new Intent(context, ConversationIntentService.class);
                 intent.putExtra(ConversationIntentService.SYNC, true);
                 if (message != null) {
-                    intent.putExtra(ConversationIntentService.AL_MESSAGE, message);
+                    intent.putExtra(ConversationIntentService.AL_MESSAGE,(Parcelable) message);
```

Comment:
@coderabbit Message will not be null. And if it is null then this check does not make sense
message != null ? (Parcelable) message : null);
because it is returing null value